### PR TITLE
LPS-116374 + LPS-116374 - React stacked stars + Fix wrong rounds in rating stars

### PR DESCRIPTION
**Pull request by boton**

### CODE REVIEW

**Comment by boton** [comment]
ci:test:sf

**Comment by boton** [comment]
ci:test:relevant

File: modules/apps/ratings/ratings-taglib/src/main/resources/META-INF/resources/js/components/Ratings.js [hunk 1]
```diff
@@ -109,4 +112,6 @@ Ratings.propTypes = {
 	url: PropTypes.string.isRequired,
 };
 
-export default Ratings;
+export default function (props) {
```
Comment (ambrinchaudhary) [inline modules/apps/ratings/ratings-taglib/src/main/resources/META-INF/resources/js/components/Ratings.js:115]: Can we simplify this and keep it the way it was? `export default Ratings;`

File: modules/apps/ratings/ratings-taglib/src/main/resources/META-INF/resources/js/components/RatingsStars.js [hunk 2]
```diff
@@ -136,7 +139,7 @@ const RatingsStars = ({
 		return Lang.sub(srAverageMessage, [averageScore, numberOfStars]);
 	};
 
-	return (
+	return type === 'stars' ? (
```
Comment (ambrinchaudhary) [inline modules/apps/ratings/ratings-taglib/src/main/resources/META-INF/resources/js/components/RatingsStars.js:142]: Would be better to use a constant/variable

File: modules/apps/ratings/ratings-taglib/src/main/resources/META-INF/resources/js/components/RatingsStars.js [hunk 3]
```diff
@@ -232,6 +235,116 @@ const RatingsStars = ({
...
+										'rate-this-x-star-out-of-x'
+								  )
+								: Liferay.Language.get(
+										'rate-this-x-stars-out-of-x'
+								  );
+						const full = label <= score;
+
+						return (
+							<Fragment key={value}>
+								<input
```
Comment (ambrinchaudhary) [inline modules/apps/ratings/ratings-taglib/src/main/resources/META-INF/resources/js/components/RatingsStars.js:295]: Awesome!

**Comment by 4lejandrito** [comment]
👍

File: modules/apps/ratings/ratings-taglib/src/main/resources/META-INF/resources/js/components/Ratings.js (hunk 1 above)
Comment (boton) [reply modules/apps/ratings/ratings-taglib/src/main/resources/META-INF/resources/js/components/Ratings.js:115]: Yep :/

**Comment by boton** [comment]
ci:forward

**Comment by boton** [comment]
ci:forward

**Pull request closed without merging**